### PR TITLE
LPS-168193 [Raylife AP] Creating the policy table

### PR DESCRIPTION
**Pull request by eliasantosf**

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

**Comment by eliasantosf** [comment]
ci:test:sf

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+	type TableContentType = {
+		[key: string]: string;
+	};
+
+	type ClaimTableType = {
+		claimStatus: {name: string};
+		dateCreated: string;
+		externalReferenceCode: string;
+		id: string;
+		r_policyToClaims_c_raylifePolicyId: any;
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx:93]: Is it possible to type it correctly?

**Comment by joanagrego** [comment]
Just starting reviewing :)

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx [hunk 2]
```diff
@@ -0,0 +1,261 @@
...
+		const policy = await getPolicyById(policyId);
+
+		return policy?.data?.productName;
+	}
+
+	async function getPolicyOwnerName(policyId: number) {
+		const policy = await getPolicyById(policyId);
+
+		return policy?.data?.policyOwnerName;
+	}
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx:106]: We have two very similar functions here. What do you think to turn it into a generic one?

File: modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx [hunk 3]
```diff
@@ -0,0 +1,261 @@
...
+		setTotalCount(totalCount - 1);
+	};
+
+	const handleEditClaim = (externalReferenceCode: string) => {
+		alert(`Edit ${externalReferenceCode} Action`);
+	};
+
+	useEffect(() => {
+		getClaims(PARAMETERS).then((results) => {
+			const claimList: TableContentType[] | any = [];
```
Comment (joanagrego) [inline modules/apps/site-initializer/site-initializer-raylife-ap/extra/remote-app/src/routes/claims/pages/ClaimsTable/index.tsx:125]: Is it possible to type it correctly? Let's try to do not to use the "any" type.

**Review by joanagrego — CHANGES_REQUESTED** [review]
Nice job! A added some suggestions :)

**Pull request closed without merging**